src/Microsoft.Identity.Client/AppConfig/AbstractApplicationBuilder.cs

```diff
@@ -381,14 +382,16 @@ public T AddKnownAuthority(Uri authorityUri, bool isDefaultAuthority)
         /// <param name="tenantId">Guid of the tenant from which to sign-in users</param>
         /// <param name="isDefaultAuthority">Boolean telling if this is the default authority
         /// for the application</param>
+        /// <param name="validateAuthority">TODO(migration): documentation</param>
         /// <remarks>You can add several authorities, but only one can be the default authority</remarks>
         /// <returns>The builder to chain the .With methods</returns>
         public T AddKnownAadAuthority(
             Uri cloudInstanceUri,
             Guid tenantId,
-            bool isDefaultAuthority)
+            bool isDefaultAuthority,
+            bool validateAuthority = false)
```

Comment:
shouldn't it be true for Aad authorities (and false for B2C authorities for example) ?

Reply:
Yes, still need to finalize this.  I will add it to NOT be available for B2C since it must be false there.  But it can be used in ADFS or AAD.  the other part of finalization is that adding multiple authorities makes no sense given this approach so I'd like to change these to WithAadAuthority, WithB2CAuthority, WithAdfsAuthority, etc and we'll then remove the "isDefaultAuthority" bit.